### ISSUE TITLE
[FEATURE] Support HubVirtualNetworkConnectionProperties ConnectivityStatus

### ISSUE THREAD

**Issue by kuwas** [Azure/azure-sdk-for-go#19521]
### Feature Request

Support `HubVirtualNetworkConnectionProperties` struct `ConnectivityStatus` field.

```
type HubVirtualNetworkConnectionProperties struct {
  ConnectivityStatus *ConnectivityStatus `json:"connectivityStatus,omitempty" azure:"ro"`
}
```

This field appears to be **undocumented** in the Azure REST API docs.
However, it's in the live response. I've tested 2 versions so far, and it's present in both:
* `2022-01-01`
* `2022-05-01`

**Comment by lirenhe** [comment]
This seems to be a Swagger correctness issue. @tadelesh, could you help to double check?

**Comment by tadelesh** [comment]
Yes. It is a swagger correctness issue. @kuwas I'll migrate this issue to the swagger repo and involve service team then.
